src/mosquito_biology.cpp

```diff
@@ -45,8 +45,8 @@ double rainfall(
     double result = g0;
     for (auto i = 0u; i < g.size(); ++i) {
         result +=
-            g[i] * cos(2 * M_PI * t * (i + 1) / 365) +
-            h[i] * sin(2 * M_PI * t * (i + 1) / 365);
+            g[i] * cos(2 * M_PI * (t % 365) * (i + 1) / 365) +
```

Comment:
are these modulos needed?

Reply:
Thanks, it was an unnecessary leftover from debugging